### DISCUSSION TITLE
THUDM/CogVideoX-5b model use

### DISCUSSION THREAD

**Discussion by vivekvp1** [THUDM/CogVideo#511] [Q&A] [answered]
Hello,
I am using the Pinokio installed version of Cog Video Studio.  
For text to image there is a choice of models THUDM/CogVideoX-5b and THUDM/CogVideoX-5b and THUDM/CogVideoX-2b.
I selected THUDM/CogVideoX-5b.  But the default is 2b.
But I am using image to video.  There is no option to select a model in image to video.  

Questions
Will it use the 5b model or 2b model?

Thank you!

V

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
这个功能并不是我们能维护到的，我觉得你应该跟 CogVideo Studio 提交问题反馈
